templates/pricing/infra.html

```diff
@@ -24,35 +26,64 @@ <h1>Open Infrastructure support & security</h1>
 <section class="p-strip--light is-shallow" id="ua-infra">
   <div class="row">
     <h3>Security maintenance and support</h3>
-    {% include "shared/pricing/_ua-for-infrastructure.html" %}
+  </div>
+  {% include "shared/pricing/_ua-for-infrastructure.html" %}
+  <div class="row">
+    <div class="col-12">
+      <p>See our consulting, architecture, deployment and training packages for <a href="/pricing/consulting#openstack">OpenStack</a>, <a href="/pricing/consulting#kubernetes">Kubernetes</a> and <a href="/pricing/consulting#kubeflow">Kubeflow</a>.</p>
+      <p>
+        <a href="/support/contact-us" class="p-button js-invoke-modal">Solve compliance and security for your Ubuntu estate</a>
+      </p>
+    </div>
   </div>
 </section>
 
 <section class="p-strip is-shallow" id="storage">
   <div class="row">
     <h3>Software-defined storage - Ceph and Swift</h3>
     <p>Only applies if attaching more than 12TB of usable Ceph/SWIFT disks (e.g 36TB raw with 3x replication) per node in the cluster. Priced by the amount exceeding the total of 12TB per node included in UA.</p>
-    {% include "shared/pricing/_ua-advanced-storage.html" %}
   </div>
+  {% include "shared/pricing/_ua-advanced-storage.html" %}
+  <div class="row">
+    <div class="col-12">
+      <p>Prices are for UA Advanced contracts. UA Standard (office hours) is at 50% discount.</p>
+      <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+    </div>
+  </div>
+
 </section>
 
 <section class="p-strip--light is-shallow" id="managed">
   <div class="row">
     <h3>Fully managed OpenStack, Kubernetes and LMA</h3>
     <p>We manage infrastructure at both the host and the guest level. Infrastructure includes Ceph and SWIFT storage, Kubernetes, OpenStack and the recommended logging, monitoring and alerting stack - <a href="/infrastructure">see what’s covered</a>.</p>
-    {% include "shared/pricing/_openstack-kubernetes-lma.html" %}
   </div>
+  {% include "shared/pricing/_openstack-kubernetes-lma.html" %}
+  <div class="row">
+    <div class="col-12">
+      <p id="k8s-per-host"><small><sup>*</sup> On a VMware or OpenStack cluster, managed guests can be priced per host in the cluster.</small></p>
+      <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+    </div>
+  </div>
+
 </section>
 
 <section class="p-strip is-shallow">
   <div class="row">
     <h3 id="maas">MAAS</h3>
     <p>Support and commercial capabilities for MAAS are included with Ubuntu Advantage Infrastructure. These additional charges apply for machines managed by MAAS and *not* covered by a UA Infrastructure contract.</p>
-    {% include "shared/pricing/_maas-pricing.html" %}
+  </div>
+  {% include "shared/pricing/_maas-pricing.html" %}
+  <div class="row">
+    <div class="col-12">
+      <p>
+        <a href="/support/contact-us" class="p-button--positive">Get in touch</a>
```

Comment:
Shouldn't this one has `js-invoke-modal`

Reply:
yes